### DISCUSSION TITLE
Cannot find .NET MAUI Blazor App (Preview)

### DISCUSSION THREAD

**Discussion by arsalanhub** [dotnet/maui#9768] [Q&A]
I am using Visual Studio Community 2022 version 17.2.5 and I cannot find above mentioned package.

**Comment by xwang-888** [comment]
Please update to version 17.3 or above.

**Reply by arsalanhub** [reply]
Currently, this package is in the preview version. Can I use it for creating apps for client-related projects?

**Reply by xwang-888** [reply]
> Visual studio 2022 17.3 is an official version, not a preview version. You can create applications through the official version of 17.3

**Reply by arsalanhub** [reply]
It is the official version but it is available only in VS code preview and not in VS code community which is my concern.
